add awareness of missing folders (#2578)
alcionai/corso: website/docs/support/known-issues.md

* Restoration of Nested attachments within Exchange Mail or Calendars aren't yet supported.

> * Folders and Calendars containing zero items or subfolders are not included in the backup.

Reviewer: ```suggestion
* Folders and Calendars containing zero items or subfolders aren't included in the backup.
```

* Provides no guarantees about whether data moved, added, or deleted in M365
  while a backup is being created will be included in the running backup.

>   Future backups run when the data isn't modified will include the data.

Reviewer: ```suggestion
  Future backups run when the data isn't modified will include the data.
  
```
